css/properties/align-content.json

```diff
@@ -237,7 +237,9 @@
               },
               "firefox": [
                 {
-                  "version_added": "52"
+                  "version_added": "52",
+                  "version_removed": "60",
+                  "notes": "As per <a href='https://drafts.csswg.org/css-align-3/#align-justify-content'>CSS Box Alignment Module Level 3</a>, <code>align-content</code> no longer accepts the <code>left</code> and <code>right</code> values"
```

Comment:
This note doesn't bring anything. Especially in the future, it will look odd.

Reply:
I've removed all the spec links, as I agree they don't make sense.